cmd/crc/cmd/daemon.go

```diff
@@ -158,6 +162,14 @@ func run(configuration *types.Configuration) error {
 	}()
 
 	c := make(chan os.Signal, 1)
+
+	if watchdog {
+		go func() {
+			_, _ = ioutil.ReadAll(os.Stdin)
```

Comment:
I added logs, it should be better.

Reply:
I don't really know how to trigger this error. Do you have an idea?

```
$ crc daemon --watchdog            
INFO listening vsock://:1024                      
INFO listening /home/guillaumerose/.crc/crc-http.sock 
^C
$ crc daemon --watchdog < /dev/null
INFO listening vsock://:1024                      
INFO listening /home/guillaumerose/.crc/crc-http.sock 
ERRO stdin is closed, shutdown...   
```

Reply:
Nope, not sure either how to trigger such a read error on stdin, I'd expect it will either never happen, or only happen in tiny corner cases. Which is why I prefer to have good logs here so that we have a hint as to what's going on if this happens, rather than assuming stdin was closed with no errors as usual.